### PR TITLE
OCPBUGS#21900: Fixes gcp CPU manager image to include more arches

### PR DESCRIPTION
**Pull request by kelbrown20**
**Version(s):** 4.14+
**Issue:** [OCPBUGS-21900](https://issues.redhat.com/browse/OCPBUGS-21900)

**Link to docs preview:**
[Using CPU manager and Topology Manager -> Setting up CPU manager](https://67221--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/using-cpu-manager#seting_up_cpu_manager_using-cpu-manager-and-topology_manager)

**QE review:**
- [x] QE has approved this change.

### CODE REVIEW

**Comment by kelbrown20** [comment]
@aleskandro Hey Alessandro! Can you PTAL at this doc bug update when you get the chance? Thank you!

**Comment by aleskandro** [comment]
/label qe-approved

Thanks @kelbrown20!

File: modules/setting-up-cpu-manager.adoc [hunk 1]
```diff
@@ -118,7 +118,7 @@ metadata:
 spec:
   containers:
   - name: cpumanager
-    image: gcr.io/google_containers/pause-amd64:3.0
+    image: gcr.io/google_containers/pause-amd64:3.2
```
Comment (aleskandro) [inline modules/setting-up-cpu-manager.adoc:121]:
```suggestion
    image: gcr.io/google_containers/pause:3.2 
```

ah, nit here (no -amd64).

**Comment by kelbrown20** [comment]
Was wondering if I could redo peer review bc of changes and was also wondering if I could get an approval for this PR with the new Travis updates. 

/remove-label peer-review-done
/label peer-review-needed

**Review by mjpytlak — APPROVED** [review]
@kelbrown20 peer review lgtm. Approved PR to allow for merge.

**Pull request merged**